### DISCUSSION TITLE
The screen is not centered when running an OpenXR game

### DISCUSSION THREAD

**Discussion by haudon** [praydog/REFramework#948] [General Questions]
When playing games such as Resident Evil 7 and Resident Evil 4 RE:2 with Virtual Desktop vDRX and running the game in OpenXR, the screen appears in the back. Is there a solution?

**Comment by armchairjedi** [comment]
Virtual Desktop Height Issue:  Its a consistent problem with VDXR and reframework.  You can go to the onscreen menu for reframework (it can be a little hard to read once VR starts unless you are in higher resolution)  select VR and click "set standing height".  This will fix it, but you have to do this every game as I have not figured out how to save the setting or modify the .config file.  With VDXR you are probably using Virtual Desktop, If you link up your headset and desktop first, then start the game with your headset on it will often start with the correct screen height.  If you use quest controllers to play, in the Virtual Desktop in game menu there is a box you can check under "emulate gamepad" that says "use Touch controllers as gamepad"  checking this box will also often reset the height, but this has to be done every time you start the game.
